resources/assets/js/components/MicrotaskingVolunteering.vue

```diff
@@ -118,15 +116,6 @@ export default {
 @import '~bootstrap/scss/variables';
 @import '~bootstrap/scss/mixins/_breakpoints';
 
-#volunteering {
```

Comment:
Has this and the breakpoint difference moved somewhere else?

Reply:
This was a bad change.  There is a duplicate id of "volunteering" which was causing a warning, because MicrotaskingPage also defines it.  But I can't see a good reason to remove the style at all.

Given that CSS can be scoped within a component there is very little reason to use id at all now, and some reasons not to.